packages/gatsby-theme-project-portal/src/components/SiteSearch.tsx

```diff
@@ -39,7 +48,8 @@ export const SiteSearch: FunctionComponent<SearchProps> = ({
           placeholder={"Type to search pages..."}
           onChange={(e) => setSearchQuery(e.target.value)}
         />
-        <div className=" pt-2">
+        <p className="px-4 text-red text-tag font-bold">{errorMessage}</p>
```

Comment:
Sorry one other comment here, do you think it's sensible for us to hide the `<p>` if there is no ErrorMessage? Given that it would show up on screen readers even if no errorMessage is present

Reply:
Yes! I'll change that, thanks Josh.